pkg/controller/migmigration/task.go

```diff
@@ -164,9 +161,6 @@ var CancelItinerary = Itinerary{
 		{phase: DeleteRestores},
```

Comment:
Once we have a solution for cleaning up pending Velero resources at migration start (https://github.com/konveyor/mig-controller/issues/703) we can also strip out the `DeleteBackups` and `DeleteRestores` phases from the `CancelItinerary`.